### DISCUSSION TITLE
How to use router?

### DISCUSSION THREAD

**Discussion by NeroBlackstone** [crewdevio/Snel#27] [Q&A] [answered]
Sorry, I just started learning svelte and deno, then I found Snel. It's an interesting idea.
But I'm confused about how to use the router, I check Svelte [official site](https://svelte.dev/faq#is-there-a-router), it says SvelteKit is the official router library.
So how to use the router with Snel? It would be better if there is a code example.
Thank you very much!

**Comment by borsemayur2** [comment, marked as answer]
@NeroBlackstone This is how routing works in snel
[Snel Routing](https://github.com/crewdevio/Snel/tree/main/core/router)

**Reply by NeroBlackstone** [reply]
Thank you very much!
